src/user/user.entity.ts

```diff
@@ -58,6 +58,18 @@ export class User {
   })
   address: string;
 
+  @Column({
+    name: 'coords_lat',
+    nullable: true,
+  })
+  coordsLat: string;
+
+  @Column({
```

Comment:
Coordinates should be required, otherwise we will be not able to schedule a travel, because we will have a user in system without necessary data.